python/pyspark/ml/wrapper.py

```diff
@@ -63,7 +64,37 @@ def _create_from_java_class(cls: Type[JW], java_class: str, *args: Any) -> JW:
         java_obj = JavaWrapper._new_java_obj(java_class, *args)
         return cls(java_obj)
 
+    def _call_remote(self, name: str, *args: Any) -> Any:
+        key = f"{self._java_obj}.{name}"
+        # TODO support args
+
+        session = SparkSession.getActiveSession()
+        client = SparkSession.getActiveSession().client
+        model_attr_command_proto = pb2.ml_pb2.MlCommand.FetchModelAttr(
+            model_ref=pb2.ml_common_pb2.ModelRef(id=key),
+            name=name
+        )
+        req = client._execute_plan_request_with_metadata()
+        req.plan.ml_command.fetch_model_attr.CopyFrom(model_attr_command_proto)
+
+        resp = client.execute_ml(req)
+
+        ret = deserialize(resp, client)
+        if resp.HasField("model_info"):
+            self._java_obj = ret
+            return None
+        if resp.HasField("is_dataframe"):
```

Comment:
Do we need to rely on servers to insert a Relation in a logic plan?

Reply:
Looks like we can get the return type by inspect if the method has type hint.

Reply:
Please leverage python annotations to get return type. It looks like the built-in PySpark ML code has type hints provided already.